src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --nav-active: #00acdc;
+  --header-color: #fff;
+  --nav-color: #000;
+  --nav-backgroundcolor: #fff;
+}
+
+* {
+  font-family:
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    'Open Sans',
+    'Helvetica Neue',
+    sans-serif;
+  font-size: 12px;
   margin: 0;
+  box-sizing: border-box;
+}
+
+.site_header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: var(--header-color);
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav li {
```

Comment:
do not add styles using combined selectors without necessary
just add class for all `li` elements and use this class to add styles

fix all similar cases